src/ImageSharp/Formats/PixelTypeInfo.cs

```diff
@@ -6,10 +6,10 @@ namespace SixLabors.ImageSharp.Formats
     /// <summary>
     /// Contains information about the pixels that make up an images visual data.
     /// </summary>
-    public class PixelTypeInfo
+    public readonly struct PixelTypeInfo
```

Comment:
Not sure if it's worth to turn this into a `struct`. The instances are not frequents, so there are no performance benefits, however it makes it easier to users to make mistakes:
Default-constructed instances are invalid (`BitsPerPixel == 0`) + it introduces the cognitive load of struct/value semantics. (Note: we plan to add more fields like a reference to the pixel type etc.)
Thoughts?

Reply:
Agreed. If this class grows, we'll want it to be a reference type. Reverting.